src/plays/netlify-card-game/NetlifyCardGame.scss

```diff
@@ -0,0 +1,129 @@
+$box-width: 50%;
+$box-height: 400px;
+$box-bg: rgba(255, 247, 247, 1);
+
+$single-box-width: 25%;
+$single-box-height: 100px;
+
+$single-item-img: url("./question.png");
+$single-item-bg: rgb(241, 239, 239);
+
+$single-item-hover: rgba(59, 154, 156, 1);
+$single-item-hover-bg: url("./question1.png");
+
```

Comment:
Add prefix for all style def based on this comment:

"Add a play specific class name here. Prefix that class name in all your style definition. It will make sure there is no conflict of the file name.

We are planning to do this change at the framework level to take care automatically. However, until that is in, please do this change."

Reply:
yup its done.